readme.txt

```diff
@@ -108,6 +108,15 @@ Web Stories are powered by [AMP](https://amp.dev/), which adds some restrictions
 
 For the plugin's full changelog, please see [the Releases page on GitHub](https://github.com/google/web-stories-wp/releases).
 
+= 1.5.0 =
+
+**Release Date:** March 18, 2021.
+
+* New feature: new Gutenberg blocks for displaying your stories.
```

Comment:
```suggestion
* New feature: new enhanced Gutenberg block for displaying your stories.
```

It's a single block. Might also want to mention the other new features of this project. So more like

```suggestion
* New feature: new enhanced Gutenberg block, widget and theme integrations for displaying your stories in creative ways.
```

Reply:
@amedina perhaps you have some good suggestion for the changelog item here?